plugins/@grouparoo/vero/__tests__/export/export-record.ts

```diff
@@ -0,0 +1,572 @@
+import "@grouparoo/spec-helper";
+import { helper } from "@grouparoo/spec-helper";
+
+import { connect } from "../../src/lib/connect";
+import { loadAppOptions, updater } from "../utils/nockHelper";
+import { VeroClient } from "../../src/lib/client";
+import { DestinationSyncModeData } from "@grouparoo/core/dist/models/Destination";
+import { exportRecord } from "../../src/lib/export/exportRecord";
+import * as readline from "readline";
+
+let client: VeroClient;
+
+let profileId = "user101";
+let profileId2 = "user102";
+let profileId3 = "user103";
+let profileId4 = "user104";
+let nonexistentprofileId = "user-not-found";
+
+const groupOne = "TEST High Value";
+const groupTwo = "TEST Spanish Speaking";
+const groupThree = "TEST Recently Added";
+
+const { newNock } = helper.useNock(__filename, updater);
+const appOptions = loadAppOptions(newNock);
+const appId = "app_a1bb05e8-0a4e-49c5-ad42-545f2e8762f9";
+
+async function cleanUp() {
+  // Clear created people
+  for (let id of [
+    profileId,
+    profileId2,
+    profileId3,
+    profileId4,
+    nonexistentprofileId,
+  ]) {
+    try {
+      await client.deleteUser(id);
+    } catch (e) {}
+  }
+}
+
+async function runExport({
+  syncOperations = DestinationSyncModeData.sync.operations,
+  oldRecordProperties,
+  newRecordProperties,
+  oldGroups,
+  newGroups,
+  toDelete,
+}) {
+  return exportRecord({
+    appOptions,
+    appId,
+    connection: null,
+    app: null,
+    destination: null,
+    destinationId: null,
+    destinationOptions: null,
+    syncOperations,
+    export: {
+      record: null,
+      recordId: null,
+      oldRecordProperties,
+      newRecordProperties,
+      oldGroups,
+      newGroups,
+      toDelete,
+    },
+  });
+}
+
+const manualCheck = async (...instructions: string[]) => {
+  if (process.env.CI) {
```

Comment:
This should probably actually be `if(newNock)` because it doesn't make sense to check unless it's really making requests. (and we don't nock on CI, so that will still work)